### DISCUSSION TITLE
相似sql查询时间差距大

### DISCUSSION THREAD

**Discussion by 1240288959** [apache/doris#6571] [A - General / Q&A]
sql A:
`SELECT 
     BITMAP_UNION_COUNT(CASE WHEN flag = 1 THEN field_id  END) AS cnt
FROM tableA`

sql B:
`SELECT 
     BITMAP_UNION_COUNT(field_id) AS cnt
FROM tableA
where flag = 1`

tableA表对field_id做了预聚合，使用explain的使用，看两个Sql都是使用了预聚合
进行查询sqlA的执行时间是sqlB的6倍，求告知sqlA执行慢的原因

**Comment by EmmyMiao87** [comment]
通过设置 is_report_success=true 获取到实际的各个算子的执行时间，具体方式参见：http://doris.apache.org/master/zh-CN/administrator-guide/running-profile.html
